universal-application-tool-0.0.1/app/controllers/PostgresController.java

```diff
@@ -46,4 +46,17 @@ public PostgresController(
             },
             httpExecutionContext.current());
   }
+
+  public Result syncAdd(String name) {
+    Person p = newPerson(name);
+    p.save();
+    return ok("person " + name + " with ID: " + p.id.toString() + " synchronously added.");
+  }
+
+  public Person newPerson(String name) {
+    Person p = new Person();
+    p.id = System.currentTimeMillis(); // not ideal, but it works
```

Comment:
The record shouldn't have an ID if it hasn't yet been persisted to the database.